R/splitBy.R

```diff
@@ -0,0 +1,181 @@
+#' Split \code{SummarizedExperiment} by grouping variables
+#'
+#' @param x A
+#'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{SummarizedExperiment}}
+#'   object
+#'
+#' @param grouping A single character value for selecting the grouping variable
+#'   from \code{colData} or \code{rowData}.
+#' 
+#' @param direction A single character value for selecting from where grouping
+#'   variable should be searched. Must be "col" or "row". 
+#'   
+#' @param ... 
+#' \itemize{
+#'   \item{\code{use_names} A boolean value to select whether to name elements of
+#'   list by their group names.}
+#' }
+#'
+#' @return
+#' List of \code{SummarizedExperiment} objects.
+#'
+#'
+#' @details
+#' \code{splitBy} split data based on grouping variable. Splitting can be done
+#' column-wise or row-wise. Returned value is a list of \code{SummarizedExperiment}
+#' objects; each element containing members of each group.
+#'
+#' @seealso
```

Comment:
Check the `@family` tag. Everything listed with that will be put under `@seealso`. Worth a try?

Reply:
I think it is possible to link only manuals with that tag. 

Meaning that it is possible to link `splitByRanks` manual page. However, it seems that it is not possible to link functions directly --> we cannot add links to `splitByRanks` and `unSplitByRanks`. We can just add link to `splitByRanks`